provider-ui/components/ProviderComponent.js

```diff
@@ -82,8 +84,6 @@ class ProviderComponent extends React.Component {
     super(props);
     this.state = {
         availableProviders: {},
-        selectedRemote: '',
```

Comment:
Why have you removed these props?

Reply:
@Nikhil-Ladha was this change with respect to setting a default provider?